File: CS/WinWebSolution.Module/Updater.cs
using System;
using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace WinWebSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(Session session, Version currentDBVersion) : base(session, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            GaugeDemoObject obj = new GaugeDemoObject(Session);
            obj.IntProperty = 50;
            obj.FloatProperty = 75.0f;
            obj.EnumProperty = Light.Green;
            obj.StringProperty = "Gauge Test1";
            obj.Save();
            GaugeDemoObject obj2 = new GaugeDemoObject(Session);
            obj2.IntProperty = 75;
            obj2.FloatProperty = 25.0f;
            obj2.EnumProperty = Light.Yellow;
            obj2.StringProperty = "Gauge Test2";
            obj2.Save();
        }
    }
}

File: VB/WinWebSolution.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl

Namespace WinWebSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal session As Session, ByVal currentDBVersion As Version)
			MyBase.New(session, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim obj As New GaugeDemoObject(Session)
			obj.IntProperty = 50
			obj.FloatProperty = 75.0f
			obj.EnumProperty = Light.Green
			obj.StringProperty = "Gauge Test1"
			obj.Save()
			Dim obj2 As New GaugeDemoObject(Session)
			obj2.IntProperty = 75
			obj2.FloatProperty = 25.0f
			obj2.EnumProperty = Light.Yellow
			obj2.StringProperty = "Gauge Test2"
			obj2.Save()
		End Sub
	End Class
End Namespace
